lib/phantomjs/core.js

```diff
@@ -328,12 +280,12 @@ function getCriticalPathCss (options) {
   debuglog('stripped out non matching media queries')
 
   page.open(options.url, function (status) {
-    if (status !== 'success') {
```

Comment:
So you mean this?

``js
if (status !== 'success') {
  errorlog('Error opening url \'' + page.reason_url + '\': ' + page.reason) // jshint ignore: line
  phantomExit(1)
}